app/scripts/lib/modal_manager.js

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'underscore',
+  'jquery',
+  'velocity',
+  'velocityui'
+], function (_, $) {
+  'use strict';
+
+  function ModalManager () {
+    this._viewStates = [];
+    this.$modal = $('#modal');
+
+    // Listen for all clicks
+    $(document).on('click', _.bind(this._closeOnClick, this));
+
+    // Close modal on resize to prevent weirdness between desktop and mobile
+    $(document).on('resize', _.debounce(_.bind(this.close, this), 100));
```

Comment:
Hmm. I wonder if flexbox gives us better odds at positioning a lightbox inside a modal vs other approaches.

This is fine for now, but it seems weird to me that you see a modal, you rotate your phone to get a better look, then it vanishes. Might be better to re-render instead of just dismiss it?

These are all details, though. I'm happy to land this as-is and polish the interactions later.